hbase-shell/src/main/ruby/shell/commands/unassign.rb

```diff
@@ -23,23 +23,18 @@ class Unassign < Command
       def help
         <<-EOF
 Unassign a region. It could be executed only when region in expected state(OPEN).
-Pass 'true' to force the unassignment ('force' will clear all in-memory state in
-master before the reassign. If results in double assignment use hbck -fix to resolve.
-To be used by experts).
 In addition, you can use "unassigns" command available on HBCK2 tool to skip the state check.
 (For more info on HBCK2: https://github.com/apache/hbase-operator-tools/blob/master/hbase-hbck2/README.md)
 Use with caution. For experts only.
 Examples:
 
   hbase> unassign 'REGIONNAME'
-  hbase> unassign 'REGIONNAME', true
   hbase> unassign 'ENCODED_REGIONNAME'
-  hbase> unassign 'ENCODED_REGIONNAME', true
 EOF
       end
 
-      def command(region_name, force = 'false')
-        admin.unassign(region_name, force)
+      def command(region_name)
```

Comment:
this breaks compatibility for the shell and will need to be release noted. Alternatively you could keep the optional argument and note that it is deprecated and ignored.

Reply:
Will fix later. Thanks.